test/e2e/framework/ingress/ingress_utils.go

```diff
@@ -400,7 +400,6 @@ func createTLSSecret(kubeClient clientset.Interface, namespace, secretName strin
 	}
 	var s *v1.Secret
 	if s, err = kubeClient.CoreV1().Secrets(namespace).Get(context.TODO(), secretName, metav1.GetOptions{}); err == nil {
-		// TODO: Retry the update. We don't really expect anything to conflict though.
```

Comment:
This TODO was left in May 2016. 
Is it really necessary to retry the update operation?
I think we remove this TODO,if neccessary i will try to do it.